lib/main_micro_alp.pm

```diff
@@ -425,7 +425,7 @@ sub load_tests {
         load_network_tests;
     } elsif (check_var('EXTRA', 'provisioning')) {
         # This module fails in MicroOS, never been run before. Need to investigate.
-        loadtest 'microos/verify_setup' unless is_microos;
+        loadtest 'microos/verify_setup';
```

Comment:
Why condition removed?

Reply:
Because this module did not work on microos before, so it was skipped. I was asked to re-introduce it and try to make it work.